src/assets/favicon.png

```diff
@@ -0,0 +1,5 @@
+<svg width="100" height="100" viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is this working properly , because your image extension is `.png` and you are embedding `.svg` code inside your file

Reply:
No, it was not. Didn't realise that as the browser was showing the previous favicon from the cache. Fixed it now👍